app/frontend/src/submissions/components/SubmissionForm/Coords.vue

```diff
@@ -508,8 +340,50 @@ export default {
         zone: null
       }
     },
+    handleDegreesChange () {
```

Comment:
Would be nice if there were unit tests for these new methods

Reply:
They are not new - just changed from the watch-ers style from before. I will try and add some unit tests to it though.

Reply:
@dainetrinidad I have added tests to the handle*Change methods.

Reply:
Thank you!